docs/FLYSKY_FRM303.md

```diff
@@ -101,4 +101,52 @@ Subsequent flashing can be done in two ways:
     - This enables programming over the USB-C port using the DFU method
     - Launch STM32CubeProgrammer and select USB as the connection method, click connect 
     - Proceed as usual
-   
+
+
+## Optional modifications ##
+
+### OLED Display Addition ###
+
+The FRM303 Tx module can be optionally equipped with a 0.96" monochrome OLED screen (128x64 pixel, SSD1306 display controller, needs I2C bus and +3.3V power supply). With the OLED, among other functions, the current link statistics data can be observed, the parameter settings of the Tx module, as well as those of a connected receiver can be altered. User interface navigation is via the 5-way joystick. The OLED can replace the CLI and the mLRS Lua script.
+
+<img src="images/FRM303-OLED-mod_finished.jpg" height="400px">
+
+To activate OLED screen, the "OLED" firmware version (indicated by a '-oled' label in the firmware file name) needs to be flashed into the Tx module.
+
+The OLED mod exploits the fact that the FRM303 module has four M2.5 threads on the front side, where the top two holes provide easy access to the interior of the housing and the bottom two holes almost perfectly match the distance of the mounting holes of a typical 0.96" OLED display (the OLED mounting holes need also to be slightly enlarged to fit the M2.5 screws).
+
+The following pictures illustrate a most simple application (click on the picture to open a bigger version):
+
+<a href="images/FRM303-OLED-mod_step1.jpg"><img src="images/FRM303-OLED-mod_step1.jpg" height="150px"></a> 
+<a href="images/FRM303-OLED-mod_step5.jpg"><img src="images/FRM303-OLED-mod_step5.jpg" height="150px"></a> 
+<a href="images/FRM303-OLED-mod_step6.jpg"><img src="images/FRM303-OLED-mod_step6.jpg" height="150px"></a> 
+<a href="images/FRM303-OLED-mod_step7.jpg"><img src="images/FRM303-OLED-mod_step7.jpg" height="150px"></a>
+<a href="images/FRM303-OLED-mod_wo_case.jpg"><img src="images/FRM303-OLED-mod_wo_case.jpg" height="150px"></a> 
+
+Optionally, you can also build a housing for the OLED screen. You can find two STL files to 3D print a mount and a housing for the OLED display under [docs/files/FRM303*.stl](https://github.com/olliw42/mLRS-docu/tree/master/docs/files)
```

Comment:
it should work using local referencing [docs/files/FRM303*.stl](files), doesn't it? or maybe ./files ... it works for files, see the link on in the top of this page as example

Reply:
Done.